src/test/java/jenkins/plugins/logstash/LogstashNotifierTest.java

```diff
@@ -32,88 +22,68 @@
 import org.mockito.Matchers;
 import org.mockito.Mock;
 import org.mockito.Mockito;
+import org.mockito.invocation.InvocationOnMock;
 import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.stubbing.Answer;
```

Comment:
There are some unused imports that slipped through your cleanup:

hudson.tasks.test.AbstractTestResultAction
org.mockito.Matchers